src/locales/en.json

```diff
@@ -33,7 +33,7 @@
     "nav_back": "Back",
 
     "field_username": "Registered email address",
-    "field_username_hint": "Enter the email address you used when you created your account",
```

Comment:
Added full stops to all hints, mainly because if an error message is also present, the screen reader reads it all as one sentence, where a small break to inform user of an error is desirable.